contracts/WildcardSteward_v2.sol

```diff
@@ -467,16 +481,59 @@ contract WildcardSteward_v2 is Initializable {
         address tokenPatron = currentPatron[tokenId];
 
         if (state[tokenId] == StewardState.Owned) {
-            uint256 totalToPayBack = price[tokenId];
-            // NOTE: pay back the deposit only if it is the only token the patron owns.
-            if (
-                totalPatronOwnedTokenCost[tokenPatron] ==
-                price[tokenId].mul(patronageNumerator[tokenId])
-            ) {
-                totalToPayBack = totalToPayBack.add(deposit[tokenPatron]);
-                deposit[tokenPatron] = 0;
-            }
+            _buy(tokenId, tokenPatron, wildcardsPercentage);
+        } else if (state[tokenId] == StewardState.Foreclosed) {
+            _buyForeclosure(tokenId);
+        }
+
+        wildcardsPercentages[tokenId] = wildcardsPercentage; //Set the percentage wildcards will recieve on next sale
```

Comment:
When the user buys, they now set the percentage that they will give to wildcards on the sale of their asset. 

Should we create a function to allow them to update this parameter like they can also update their price? Probably not necessary imo.